src/App.tsx

```diff
@@ -1,27 +1,66 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const handleAllClick = async () => {
+    try {
+      setGoods(await getAll());
+    } catch (error) {
```

Comment:
<img width="1337" alt="Screenshot 2023-03-13 at 17 08 43" src="https://user-images.githubusercontent.com/53228259/224743261-5babbbe8-d5bc-41aa-b0cc-bef830a434cf.png">

all your catches aren't working
<img width="328" alt="Screenshot 2023-03-13 at 17 09 26" src="https://user-images.githubusercontent.com/53228259/224743484-9232c956-2d90-4454-847a-b2dade065650.png">

also add catch to api